src/coreclr/jit/llvm.cpp

```diff
@@ -832,6 +832,30 @@ void Llvm::buildAdd(GenTree* node, Value* op1, Value* op2)
     }
 }
 
+void Llvm::buildDiv(GenTree* node, Value* op1, Value* op2)
```

Comment:
This should be `buildDiv(node)` and consume the operands as their actual types inside the method.

Right now, I suspect, it will not handle `DIV(short, byte)`-like constructs well.